presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -828,4 +828,70 @@ private static Double mapL2Norm(Block map)
 
         return Math.sqrt(norm);
     }
+
+    @ScalarFunction
+    @Description("binomial confidence interval lower bound using Wilson score")
+    @Nullable
+    @SqlType("double")
+    public static Double wilsonIntervalLower(@SqlType(StandardTypes.DOUBLE) double pos, @SqlType(StandardTypes.DOUBLE) double neg, @SqlType(StandardTypes.DOUBLE) double z)
+    {
+        if (pos < 0 || neg < 0 || z < 0) {
+            return null;
```

Comment:
This should be an error rather than returning null:
```java
checkCondition(pos >= 0, INVALID_FUNCTION_ARGUMENT, "positive observation count must not be negative");
...